editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -940,36 +946,79 @@ void Polygon2DEditor::_uv_input(const Ref<InputEvent> &p_input) {
 			uv_edit_draw->queue_redraw();
 		}
 	}
+}
 
-	Ref<InputEventMagnifyGesture> magnify_gesture = p_input;
-	if (magnify_gesture.is_valid()) {
-		uv_zoom->set_value(uv_zoom->get_value() * magnify_gesture->get_factor());
+void Polygon2DEditor::_center_view() {
+	Size2 texture_size = Size2();
```

Comment:
```suggestion
	Size2 texture_size;
```
Unnecessary default value